test/built-ins/Function/prototype/toString/AsyncGenerator.js

```diff
@@ -0,0 +1,16 @@
+// Copyright 2017 André Bargull. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+esid: pending
+description: >
+  Function.prototype.toString on an async generator created with the
+  AsyncGenerator constructor.
+features: [async-iteration]
+---*/
```

Comment:
This could be problematic: https://github.com/tc39/Function-prototype-toString-revision doesn't cover async generator functions and https://github.com/tc39/proposal-async-iteration doesn't cover the Function.prototype.toString changes. 😕